config/ember-try.js

```diff
@@ -51,6 +51,7 @@ module.exports = function() {
           },
         },
         {
+          allowedToFail: true,
```

Comment:
It seemed like `ember-basic-dropdown` had an issue with the canary version of `v3.25.0`.

https://github.com/ember-learn/guidemaker-ember-template/runs/1436305327?check_suite_focus=true